docs/running-on-yarn.md

```diff
@@ -492,6 +492,13 @@ To use a custom metrics.properties for the application master and executors, upd
   If it is not set then the YARN application ID is used.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.shuffle.service.name</code></td>
```

Comment:
I added this option under *Spark Properties* section and not under *Configuring the External Shuffle Service* section because it's a "client" setting, not the setting *of* the external shuffle service.